File: Java/백준/Bronze/28701. 세제곱의 합/세제곱의 합.java
import java.util.Scanner;

public class Main {
    public static void main (String[] args) {
        Scanner sc = new Scanner (System.in);
        
        int n = sc.nextInt();
        
        long sum = n * (n + 1) / 2;
        long sumSquared = sum * sum;
        
        System.out.println(sum);
        System.out.println(sumSquared);
        System.out.println(sumSquared);
        
        sc.close();
    }
}

File: C++17/백준/Bronze/28701. 세제곱의 합/세제곱의 합.cc
#include <iostream>
using namespace std;

int main() {
    long long n;
    cin >> n;

    long long sum = n * (n + 1) / 2;
    long long sumSquared = sum * sum;

    cout << sum << "\n";
    cout << sumSquared << "\n";
    cout << sumSquared << "\n";

    return 0;
}